flakey: TestEvents

```
--- FAIL: TestEvents (0.41s)
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-controller-7bd96d8c6f-w7qbv] Message: [Liveness probe failed: Get http://10.244.0.5:9995/ping: dial tcp 10.244.0.5:9995: connect: connection refused]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-controller-7bd96d8c6f-w7qbv] Message: [Readiness probe failed: Get http://10.244.0.5:9995/ready: dial tcp 10.244.0.5:9995: connect: connection refused]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-destination-9d6454bcd-zrhsz] Message: [Readiness probe failed: HTTP probe failed with statuscode: 503]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-destination-9d6454bcd-zrhsz] Message: [Liveness probe failed: Get http://10.244.0.6:9996/ping: dial tcp 10.244.0.6:9996: connect: connection refused]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-destination-9d6454bcd-zrhsz] Message: [Liveness probe failed: Get http://10.244.0.6:4191/metrics: dial tcp 10.244.0.6:4191: connect: connection refused]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-grafana-6cb47d47b6-9mj4z] Message: [Readiness probe failed: HTTP probe failed with statuscode: 503]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-prometheus-dc747dc74-bbx4s] Message: [Readiness probe failed: HTTP probe failed with statuscode: 503]
    install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-sp-validator-6476b7c69f-bfg5j] Message: [Readiness probe failed: HTTP probe failed with statuscode: 503]
     install_test.go:605: Found known warning event: Reason: [Unhealthy] Object: [linkerd-web-bc689f779-jzpvc] Message: [Readiness probe failed: HTTP probe failed with statuscode: 503]
    install_test.go:612: Found unexpected warning events:
        Reason: [FailedKillPod] Object: [linkerd-prometheus-dc747dc74-bbx4s] Message: [error killing pod: failed to "KillPodSandbox" for "78095b87-2827-451a-9f53-67f01053839f" with KillPodSandboxError: "rpc error: code = Unknown desc = failed to destroy network for sandbox \"8e99693d3649f3b59d3b51415a77758ac42f3b17f6cc2ed8ee6146837e0a781b\": could not teardown ipv6 dnat: running [/usr/sbin/ip6tables -t nat -X CNI-DN-b14d1c5a017e5ba9d88da --wait]: exit status 1: ip6tables: No chain/target/match by that name.\n"
        ]
```

--- a/test/install_test.go
+++ b/test/install_test.go
@@ -88,7 +88,7 @@ var (
 		`(Liveness|Readiness) probe failed: Get http://.*: read tcp .*: read: connection reset by peer`,
 		`(Liveness|Readiness) probe failed: Get http://.*: net/http: request canceled .*\(Client\.Timeout exceeded while awaiting headers\)`,
 		`Failed to update endpoint .*-upgrade/linkerd-.*: Operation cannot be fulfilled on endpoints "linkerd-.*": the object has been modified; please apply your changes to the latest version and try again`,
-		`error killing pod: failed to "KillPodSandbox" for ".*" with KillPodSandboxError: "rpc error: code = Unknown desc = failed to destroy network for sandbox \\".*\\": could not teardown ipv4 dnat: running \[/usr/sbin/iptables -t nat -X CNI-DN-.* --wait\]: exit status 1: iptables: No chain/target/match by that name\.\\n"`,
+		`error killing pod: failed to "KillPodSandbox" for ".*" with KillPodSandboxError: "rpc error: code = Unknown desc = failed to destroy network for sandbox \\".*\\": could not teardown (ipv4|ipv6) dnat: running \[/usr/sbin/(iptables|ip6tables) -t nat -X CNI-DN-.* --wait\]: exit status 1: (iptables|ip6tables): No chain/target/match by that name\.\\n"`,
 	}, "|"))
 
 	injectionCases = []struct {

